### PR TITLE
Add TypeScript definition file

### PR DESCRIPTION
**Pull request by svbutko**

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+declare module "react-native-hyperlinked-text" {
+
+    import {TextStyle} from "react-native";
+
+    export interface ILinkDef {
+        regex: RegExp;
+        style?: TextStyle;
+        replaceText?: (orig: string, text: string, url: string) => string;
+        onPress?: (orig: string, text: string, url: string) => void;
```
Comment (daramasala) [inline index.d.ts]: Also we have `noPress` property. An optional `boolean` that lets you override the global press handler with no handling.

File: index.d.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    }
+
+    interface IProps {
+        style?: TextStyle;
+        linkDefs?: ILinkDef[];
+
```
Comment (daramasala) [inline index.d.ts]: Here we have also `onLinkPress` and `linkStyle` that let you define global press handler and style for links.

**Comment by daramasala** [comment]
Thanks for your contribution! Please see my comments.

**Comment by svbutko** [comment]
Hi, thank you, I'll add and test those today.

File: index.d.ts (hunk 1 above)
Comment (daramasala) [reply index.d.ts]: ?

**Comment by daramasala** [comment]
Thank you very much for your contribution!

**Pull request merged**